myfirstproject/index.html

```diff
@@ -0,0 +1,93 @@
+    </article>
+
+    <blockquote>
+        <p>SOME CHILDREN COULD CREATIVELY ENJOY INVENTING FUNCTIONS</p>
+    </blockquote>
+
+    <article>
+        <h2>Programming</h2>
+    </article>
+    <article>
+        <p>Very soon the children were able to construct their own rod calculator in Haskell, by first constructing the engine using the Colour data type. Then the children asked the same role-play questions, this time by typing the language into the interpreter window. It soon became apparent that errors in the engine meant that the interpreter would not run, and that this would require carefully debugging and retyping. The children could spot bugs in the engine because they had already worked out the answer. The staircase was a familiar pattern, so they could predict what the output should be. Children were also able to edit the Colour data type to model odd or even staircases and test their functions to see what the effect would be on the derived successor function. A transcript (see insert) shows a pupil working with the number names for the rods, measured with White rather than the colour names. He becomes aware as he is speaking that the version of the succ function that works for a staircase with a step size White is distinct from the version of succ in an even staircase, and he corrects himself. It is clear to see that he has constructed an image of the staircase and is virtually traversing it.</p>
+    </article>
+    <figure>
+        <img src="../myfirstproject/img/2.jpg" alt="math">
+        <figcaption>Transcript of a conversation about program behaviour.</figcaption>
+    </figure>
+    <aside>
+        <h2>Where to go next</h2>

```

Comment:
h2 here and h2 used before are different a bit. That means that you should not use h2 for the previous titles and this one